xiaomi_gateway/__init__.py

```diff
@@ -42,11 +44,12 @@ def discover_gateways(self):
         if self._interface != 'any':
             _socket.bind((self._interface, 0))
 
+        discovery_retries = DEFAULT_DISCOVERY_RETRIES
```

Comment:
What about remove this line and use DEFAULT_DISCOVERY_RETRIES here:
https://github.com/Danielhiversen/PyXiaomiGateway/pull/149/files#diff-2faa4e79e6562a2399d8a55fb970fd4cR111

Reply:
This was @karlkar's solution. Sorry for the noise. My preference is a available local variable for the complete code path optionally overriden by some HA configuration (in future!).